.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -7,30 +7,18 @@ assignees: ''
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+**The problem(s)**
 
-**To Reproduce**
 
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+**To Reproduce**
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+**Expected behavior(s)**
 
 **Screenshots**
-If applicable, add screenshots to help explain your problem.
 
 **Thyra version**
 The version of Thyra the bug is present in.
 
 **Desktop (please complete the following information):**
  - OS: [e.g. MacOS, Windows...]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
-
-**Additional context**
-Add any other context about the problem here.
\ No newline at end of file
+ - Any other useful info
```

Comment:
Since you merged the `Additional context` section into the `Desktop` section, I think we should rename it to `Additional context` since it includes the OS and any other information de user has to give us.

Reply:
I'm generally not a huge fan of the "fourre tout" categories. But you had a point that it's not relevant into the `Desktop` section. So I modified it as you suggested